script.js

```diff
@@ -1,12 +1,370 @@
+  searchInput.classList.add("search-input", "form-control");
+  searchInput.placeholder = "Search for your favorite show here";
+
+  let searchBarContainer = document.createElement("section");
+  searchBarContainer.id = "searchbar-container";
+  searchBarContainer.classList.add("col-4", "my-2");
+  searchBarContainer.appendChild(searchInput);
+
+  let label = document.createElement("h6");
+  label.id = "label-show";
+  label.classList.add("col-2", "my-2", "mt-3");
+  label.textContent = "Search here:";
+
+  let searchContainer = document.getElementById("search-container");
+  searchContainer.append(label, searchBarContainer);
+
+  container.appendChild(searchContainer);
+  searchResult.classList.add("search-result");
+
+  searchInput.addEventListener("keyup", (event) => {
+    let searchString = event.target.value.toLowerCase();
+
+    let filteredShows = allShows.filter((show) => {
+      return (
+        show.name.toLowerCase().includes(searchString) ||
+        show.summary.toLowerCase().includes(searchString)
+      );
+    });
+    searchResult.innerHTML = `Displaying ${filteredShows.length}/${allShows.length}`;
+
+    container.appendChild(searchResult);
+    document.getElementById("container").remove();
+    makePageForShows(filteredShows);
+  });
+}
+
+function makeDropdownForShows() {
+  let container = document.getElementById("root");
+
+  const allShows = getAllShows();
+  sortShowsByName(allShows);
+
+  let select = document.createElement("select");
+  select.textContent = "Test";
+  select.classList.add("form-control");
+
+  let defaultOption = document.createElement("option");
+  defaultOption.innerHTML = "All Shows";
+  defaultOption.value = 0;
+  select.appendChild(defaultOption);
+
+  let option;
+  for (let show of allShows) {
+    option = document.createElement("option");
+    option.innerHTML = `${show.name}`;
+    option.value = show.id;
+    select.appendChild(option);
+  }
+  let dropMenuContainer = document.createElement("section");
+  dropMenuContainer.classList.add("col-4", "my-2");
+  dropMenuContainer.appendChild(select);
+
+  let searchContainer = document.createElement("section");
+  searchContainer.classList.add("row");
+  searchContainer.id = "search-container";
+
+  let label = document.createElement("h6");
+  label.classList.add("col-2", "my-2", "mt-3");
+  label.textContent = "Choose show:";
+
+  searchContainer.append(label, dropMenuContainer);
+  container.appendChild(searchContainer);
+
+  select.addEventListener("change", (event) => {
+    if (event.currentTarget.value == 0) {
+      document.getElementById("select-episode-element").remove();
+      document.getElementById("episode-searchbar").remove();
+      makeSearchForShows();
+      const allShows = getAllShows();
+      makePageForShows(allShows);
+      searchResult.innerHTML = `Displaying ${allShows.length}/${allShows.length}`;
+    } else {
+      showSearchBarExist = document.getElementById("searchbar-container");
```

Comment:
I've noticed you've got a bug where if you repeatedly choose tv shows from the drop-down, you keep adding a new `episode-searchbar` element. You need to make sure that you remove the parent elements of `episode-searchbar` and `select-episode-element` since you recreate them on line 345.

Also, if I select the `All shows` options, you need to remove the parent elements of `episode-searchbar` and `select-episode-element` as well.